### PR TITLE
Streamz gQuant example

### PR DESCRIPTION
**Pull request by yidong72**
In this PR, it adds a dynamic gQuant nodes registration feature so the developer can build up a new set of gQuant nodes one by one. The node can be used properly in the client without server restart and browser refresh. It adds a cuStreamz gQuant example and updates the old 05_customize_nodes_with_ports notebook.

### CODE REVIEW

**Comment by GPUtester** [comment]
Please update the changelog in order to start CI tests.

View the gpuCI docs [here](https://docs.rapids.ai/gpuci#open-pr-or-update-pr).

**Review by avolkov1 — CHANGES_REQUESTED** [review]
Please squash the commits.
```
git reset --soft a6b5d2c && git add -u && git commit --amend --no-edit

# you should now have just one commit after 6b7aa35
git rebase -i 6b7aa35

# choose reword and give that single commit some descriptive message.
# then: git push --force

```

After squashing the branch appeared like this for me:
```
commit da217bb560ce43598c174c772696cf067df845b5
Author: Yi Dong <yidong@nvidia.com>
Date:   Wed May 13 16:28:45 2020 -0700

    Add streamz examples
    
    Register gQuant Nodes dynamically with gQuantLab.

commit 6b7aa35faa2cb6916750a9fd22d4d315751bb85f
Author: yidong72 <43824965+yidong72@users.noreply.github.com>
Date:   Mon Oct 12 16:25:44 2020 -0400

    Nemo and xgboost integration  (#103)
    
    * Add NeMo and Cuml examples
    * clean up the dask client
    * Handle non-dask inputs for delayed dask processing.
    * Refactor NeMo gQuant nodes into a gquant plugin module.
.
.
.
```

I'll make a PR to fix the columns validation.

Thanks

**Comment by yidong72** [comment]
> Please squash the commits.
> 
> ```
> git reset --soft a6b5d2c && git add -u && git commit --amend --no-edit
> 
> # you should now have just one commit after 6b7aa35
> git rebase -i 6b7aa35
> 
> # choose reword and give that single commit some descriptive message.
> # then: git push --force
> ```
> 
> After squashing the branch appeared like this for me:
> 
> ```
> commit da217bb560ce43598c174c772696cf067df845b5
> Author: Yi Dong <yidong@nvidia.com>
> Date:   Wed May 13 16:28:45 2020 -0700
> 
>     Add streamz examples
>     
>     Register gQuant Nodes dynamically with gQuantLab.
> 
> commit 6b7aa35faa2cb6916750a9fd22d4d315751bb85f
> Author: yidong72 <43824965+yidong72@users.noreply.github.com>
> Date:   Mon Oct 12 16:25:44 2020 -0400
> 
>     Nemo and xgboost integration  (#103)
>     
>     * Add NeMo and Cuml examples
>     * clean up the dask client
>     * Handle non-dask inputs for delayed dask processing.
>     * Refactor NeMo gQuant nodes into a gquant plugin module.
> .
> .
> .
> ```
> 
> I'll make a PR to fix the columns validation.
> 
> Thanks

I squashed the commits using your commands.

File: gquant/dataframe_flow/taskGraph.py [hunk 1]
```diff
@@ -10,9 +10,27 @@
 import copy
 import traceback
 import dask
+import cloudpickle
```
[the remark this answers is not in the document]
Comment (yidong72) [reply gquant/dataframe_flow/taskGraph.py:13]: added it explicitly.

File: notebooks/05_customize_nodes_with_ports.ipynb [hunk 2]
```diff
@@ -245,6 +254,39 @@
     "        return output"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's register the dynamically defined gQuant nodes so the client knows about them "
```
[the remark this answers is not in the document]
Comment (yidong72) [reply notebooks/05_customize_nodes_with_ports.ipynb:261]: done

File: notebooks/05_customize_nodes_with_ports.ipynb [hunk 3]
```diff
@@ -380,18 +466,9 @@
...
-    "outlist = [\n",
-    "    'points_task.points_df_out',\n",
-    "    'distance_by_cudf.distance_df',\n",
-    "    'distance_by_cudf.distance_abs_df'\n",
-    "]\n",
-    "r = task_graph.run(outputs=outlist)\n",
+    "r = task_graph.run()\n",
     "points_df = r['points_task.points_df_out']\n",
-    "task_graph.run(outputs=outlist, formated=True)"
+    "task_graph.run(formated=True)"
```
[the remark this answers is not in the document]
Comment (yidong72) [reply notebooks/05_customize_nodes_with_ports.ipynb:471]: removed

File: gquant/dataframe_flow/taskGraph.py [hunk 4]
```diff
@@ -10,9 +10,27 @@
...
+import base64
+from types import ModuleType
+from .util import get_encoded_class
 
 __all__ = ['TaskGraph', 'OutputCollector']
```
[the remark this answers is not in the document]
Comment (yidong72) [reply gquant/dataframe_flow/taskGraph.py:19]: fixed in the other PR

**Pull request merged**